src/App.tsx

```diff
@@ -1,16 +1,24 @@
+import { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import { NewMovie } from './components/NewMovie';
 import moviesFromServer from './api/movies.json';
+import { Movie } from './types/Movie';
 
 export const App = () => {
+  const [movies, setMovies] = useState([...moviesFromServer]);
+
+  const handleAddingMovie = (movie: Movie) => {
```

Comment:
```suggestion
  const handleMovieAddition = (movie: Movie) => {
```
Just a suggestion of a more appropriate naming